core/storage/exploration/gae_models.py

```diff
@@ -239,6 +239,61 @@ def has_reference_to_user_id(cls, unused_user_id):
         return False
 
 
+class ExplorationMathRichTextInfoModel(base_models.BaseModel):
+    """Temporary Storage model for storing information useful while generating
+    images for math rich-text components in explorations.
+
+    The id of each instance is the id of the corresponding exploration.
+    """
+
+    # A boolean which indicates whether the exploration requires images to be
+    # generated and saved for the math rich-text components. If this field is
+    # False, we will need to generate math rich-text component images for the
+    # exploration. The field will be true only if for each math rich-text
+    # components there is a valid image stored in the datastore.
+    math_images_generation_required = ndb.BooleanProperty(
+        indexed=True, default=True)
+    # Approximate maximum size of Math rich-text components SVG images that
+    # would be generated for the exploration according to the length of
+    # raw_latex value.
+    estimated_max_size_of_images_in_bytes = ndb.IntegerProperty(
+        indexed=True, default=0)
+    # Set of latex values from all the math-rich text components of the
+    # exploration.
+    latex_values = ndb.JsonProperty(default=None)
+
+    @staticmethod
+    def get_deletion_policy():
+        """ExplorationMathRichTextInfoModel are temporary model that will be
+        deleted after user migration.
+        """
+        return base_models.DELETION_POLICY.DELETE
+
+    @staticmethod
+    def get_export_policy():
+        """Model does not contain user data."""
+        return base_models.EXPORT_POLICY.NOT_APPLICABLE
+
+    @classmethod
+    def get_math_exploration_count(cls):
+        """Returns the total number of math explorations."""
```

Comment:
"math exploration" is not a thing, and it could be construed to mean "explorations in the math category". Please describe this using more precise terms.

Reply:
done